email/src/main/java/edu/jhuapl/dorset/demos/EmailManager.java

```diff
@@ -0,0 +1,393 @@
+package edu.jhuapl.dorset.demos;
+
+import java.io.IOException;
+import java.util.ArrayDeque;
+import java.util.Properties;
+
+import javax.mail.Flags;
+import javax.mail.Folder;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Multipart;
+import javax.mail.Part;
+import javax.mail.Session;
+import javax.mail.Store;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigValue;
+
+public class EmailManager {
+    
+    private static final Logger logger = LoggerFactory.getLogger(EmailManager.class);
+    
+    private static final String USERNAME_KEY = "username";
+    private static final String PASSWORD_KEY = "password";
+    private static final String MAIL_STORE_TYPE_KEY = "mailStoreType";
+    private static final String HOST_KEY = "host";
+
+    private String user;
+    private String password;
+    private String mailStoreType;
+    private String host;
+    
+    private Session session;
+    private Store store;
+    private Folder inboxFolder;
+    private Folder errorFolder;
+    private Folder completeFolder;
+    private Message[] unreadMessages = new Message[1];
+    private Message[] processingMessages = new Message[1];
+    private ArrayDeque<Message> handler = new ArrayDeque<Message>();
+    private static final int HOLDS_MESSAGES = 1;
+    private String response = "";
+    
+    /**
+    * EmailManager Constructor
+    * 
+    * @param config  Configuration object that stores mail server information, username, and password
+    */
+    public EmailManager(Config config) {
+        
+        this.user = config.getString(USERNAME_KEY);
+        this.password = config.getString(PASSWORD_KEY);
+        this.mailStoreType = config.getString(MAIL_STORE_TYPE_KEY);
+        this.host = config.getString(HOST_KEY); 
+        Properties prop = extractProperties(config);
+        
+        try {
+            session = Session.getDefaultInstance(prop);
+            store = session.getStore(mailStoreType);
+            store.connect(host, this.user, this.password);
```

Comment:
You use "this." for user and password but not for host. You should be consistent.